step-templates/mysql-add-database-user-to-role.json

```diff
@@ -4,13 +4,13 @@
     "Description": "Adds a database user to a role",
     "Author": "twerthi",
     "ActionType": "Octopus.Script",
-    "Version": 6,
+    "Version": 8,
```

Comment:
7?

Reply:
Bawhaaha!  I was updating multiple templates and forgot if I'd already incremented it apparently :P  Nice catch ;)